include/deal.II/base/mpi.h

```diff
@@ -1071,6 +1071,25 @@ namespace Utilities
            const T &          object_to_send,
            const unsigned int root_process = 0);
 
+    /**
```

Comment:
```suggestion
    /**
     * Sends an object @p object_to_send from the process @p root_process
     * to all other processes.
     *
```
I'd suggest to describe what this function actually does in the very first sentence.